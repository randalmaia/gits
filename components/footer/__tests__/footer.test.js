import React from 'react'
import renderer from 'react-test-renderer'

import Footer from '../footer.js'

describe('On render Footer', () => {
  it('shows the Footer layout', () => {
    const component = renderer.create(<Footer></Footer>)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})