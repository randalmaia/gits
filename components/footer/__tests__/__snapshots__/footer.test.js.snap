// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`On render Footer shows the Footer layout 1`] = `
<footer
  className="jsx-53131117 footer"
>
  <div
    className="jsx-53131117 row center-xs"
  >
    <div
      className="jsx-53131117 col-xs-2"
    >
      <div
        className="jsx-53131117 box footer-message"
      >
        <a
          className="jsx-53131117"
          href="http://twitter.com/randalmaia"
          target="_blank"
        >
          made by randalmaia
        </a>
         ❤️
      </div>
    </div>
  </div>
</footer>
`;
